{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -389,9 +388,6 @@ def filter(self, record):
     LOGGING["loggers"]["django"]["handlers"].append("rollbar")
     LOGGING["loggers"]["{{ cookiecutter.project_slug }}"]["handlers"].append("rollbar")
 
-# Popular testing framework that allows logging to stdout while running unit tests
-TEST_RUNNER = "django_nose.NoseTestSuiteRunner"
```

Comment:
This is the most impactful change. Django-nose is not compatible with 3.12, and its maintenance status is questionable, so I think it's best to remove it. I don't think it will make a huge difference to our dev and CI workflows, but let me know if I'm missing something.

Reply:
thank you for starting this PR! 🔥

Reply:
testing this on a project now. So far this seems to be a change that doesn't matter